LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2018 Salt Stack Formulas
```

Comment:
This is not correct, copyright holder is Salt Formulas Community as in debian/copyright or Mirantis, Inc. et al

Reply:
I took it form here: https://github.com/salt-formulas/salt-formula-apache/blob/master/LICENSE
I'll change it...

Reply:
Fixed...